docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -315,6 +319,10 @@ KubeadmConfigTemplate `spec.template.spec` has been aligned to changes in the [K
     however, using different timeouts for init and join will be enabled only when v1beta1 is removed.
   - `spec.kubeadmConfigSpec.joinConfiguration.discovery.timeout` field has been removed. Use `spec.kubeadmConfigSpec.joinConfiguration.timeouts.tlsBootstrapSeconds` instead.
 - The `spec.kubeadmConfigSpec.useExperimentalRetryJoin` field (deprecated in CAPI v1.2!) has been removed.
+- The following `spec.kubeadmConfigSpec` fields have been removed because not necessary (Cluster API automatically applies the right gvk when generating kubeadm config files):
```

Comment:
```suggestion
- The following `spec.kubeadmConfigSpec` fields have been removed because they are not necessary (Cluster API automatically applies the right gvk when generating kubeadm config files):
```